src/connectors/hoopladigital.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+Connector.playerSelector = '.duration-1000';
+
+Connector.trackSelector = '.text-current > h2';
+
+Connector.getTrackInfo = () => {
+	const artistAlbumText = Util.getTextFromSelectors('.text-current > p');
+
+	const lastSeparator = artistAlbumText.lastIndexOf(' - ');
```

Comment:
```suggestion
	const lastSeparatorIndex = artistAlbumText.lastIndexOf(' - ');
```

Could be renamed for clarity but not a blocker